src/puppetlabs/puppetdb/http/event_counts.clj

```diff
@@ -9,27 +9,23 @@
             [net.cgrand.moustache :refer [app]]))
 
 (defn routes
-  [version]
-  (app
-   [""]
-   {:get (fn [{:keys [params globals paging-options]}]
-           (let [{:strs [query summarize_by counts_filter count_by] :as query-params} params
-                 query-options (merge {:counts_filter (if counts_filter (json/parse-strict-string counts_filter true))
-                                       :count_by count_by}
-                                      (events-http/validate-distinct-options! query-params))]
-             (produce-streaming-body
-              :event-counts
-              version
-              query
-              [summarize_by query-options paging-options]
-              (:scf-read-db globals)
-              (:url-prefix globals))))}))
+  [globals]
+  (let [{:keys [api-version url-prefix scf-read-db]} globals]
+   (app
+    [""]
+    {:get (fn [{:keys [params paging-options]}]
+            (let [{:strs [query summarize_by counts_filter count_by] :as query-params} params
+                  query-options (merge {:counts_filter (some-> counts_filter (json/parse-strict-string true))
```

Comment:
Aren't they the same thing in these cases? Is it just the case that `some` is unfamiliar?